cdap-docs/examples-manual/source/examples/clicks-and-views.rst

```diff
@@ -0,0 +1,147 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform Clicks and Views Application
+    :copyright: Copyright © 2016 Cask Data, Inc.
+
+.. _examples-clicks-and-views:
+
+================
+Clicks and Views
+================
+
+A Cask Data Application Platform (CDAP) example demonstrating a reduce-side join
+across two streams, using MapReduce.
+
+
+Overview
+========
+This application has a MapReduce which processes records in two streams and outputs to a PartitionedFileSet.
+The ``ClicksAndViewsMapReduce`` processes the records from a ``views`` stream and a ``clicks`` stream,
+then joining on the ``viewId`` of the records and writing the joined records to a ``joined`` PartitionedFileSet.
+
+Let's look at some of these components, and then run the application and see the results.
+
+The Clicks and Views Application
+--------------------------------
+As in the other :ref:`examples <examples-index>`, the components
+of the application are tied together by the class ``ClicksAndViews``:
+
+.. literalinclude:: /../../../cdap-examples/ClicksAndViews/src/main/java/co/cask/cdap/examples/clicksandviews/ClicksAndViews.java
+   :language: java
+   :lines: 29-
+
+Data Storage
+------------
+- *views* input``Stream`` contains ad views, representing an advertisement downloaded to a viewer's screen.
+- *clicks* input ``Stream`` contains ad clicks, representing when a viewer clicks on an advertisement.
+- *joined* output ``PartitionedFileSetFileSet`` contains the joined ad views, which additionally contains the
+  count of clicks each ad view has. This Dataset is partitioned on the logical start time of the MapReduce.
+
+MapReduce over Multiple Inputs
+------------------------------
+``ClicksAndViewsMapReduce`` is a MapReduce that reads from the *clicks* and *views* ``Stream``s and writes to
+the *joined* ``PartitionedFileSet``. The ``beforeSubmit`` method prepares the MapReduce program. It sets up the
+two ``Stream``s as input and sets up the ``PartitionedFileSet`` as output, with the appropriate ``PartitionKey``.
+The Mapper class then keys each of the records based upon the ``viewId``. That makes it so that all clicks and views
+for a particular ``viewId`` go to a single Reducer for joining.
+
+
+.. Building and Starting
+.. =====================
+.. |example| replace:: ClicksAndViews
+.. |example-italic| replace:: *ClicksAndViews*
+.. |application-overview-page| replace:: :cdap-ui-apps-programs:`application overview page, programs tab <ClicksAndViews>`
+
+.. include:: _includes/_building-starting-running.txt
+
+
+Running the Example
+===================
+
+Ingesting Records
+-----------------
+Begin by uploading a file containing some newline-separated records representing view events into the *views* stream:
+
+.. tabbed-parsed-literal::
+
+  $ cdap-cli.sh load stream views examples/ClicksAndViews/resources/views.txt
+  
+  Successfully sent stream event to stream 'views'
```

Comment:
That's what is printed by LoadStreamCommand. I agree though, it seems like an improper message.
How about "Successfully loaded file to stream 'views'"?